lib/src/components/servo/service.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:grpc/grpc.dart';
+import 'package:viam_sdk/src/components/servo/servo.dart';
+import 'package:viam_sdk/src/gen/common/v1/common.pb.dart';
+import 'package:viam_sdk/src/gen/component/servo/v1/servo.pbgrpc.dart';
+import 'package:viam_sdk/src/resource/manager.dart';
+import 'package:viam_sdk/src/utils.dart';
+
+class ServoService extends ServoServiceBase {
+  final ResourceManager _manager;
+
+  ServoService(this._manager);
+
+  Servo _servoFromManager(String name) {
+    try {
+      return _manager.getResource(Servo.getResourceName(name));
+    } catch (e) {
+      // TODO make rescource not found GRPC error
+      print('Exception details:\n $e');
+      rethrow;
+    }
+  }
+
+  @override
+  Future<MoveResponse> move(ServiceCall call, MoveRequest request) async {
+    Servo servo = _servoFromManager(request.name);
+    await servo.move(request.angleDeg, extra: request.extra.toMap());
+    return MoveResponse();
+  }
+
+  @override
+  Future<DoCommandResponse> doCommand(ServiceCall call, DoCommandRequest request) async {
+    // TODO: implement doCommand
+    throw UnimplementedError();
```

Comment:
servo should have a docommand - was it not generated?

Reply:
I was putting off implementing it because there would be some extra work required to get doCommand working. I've done that now and included the changes in this PR.